src/less/chip/chip.less

```diff
@@ -0,0 +1,61 @@
+@import "../variables/variables.less";
+
+.chip {
+    align-items: center;
+    background-color: var(--color-background-tertiary);
+    border-radius: @spacing-400;
+    color: var(--color-foreground-primary);
+    display: inline-flex;
+    font-size: @font-size-14;
+    gap: 2px;
+    justify-content: space-between;
+    max-width: 296px;
+    min-height: @spacing-250;
+    min-width: 32px;
+    padding: 0 @spacing-150;
+}
+
+.chip__text {
+    overflow: hidden;
+    padding: @spacing-75 0;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+button.chip__button {
+    background-color: transparent;
+    border: none;
+    border-radius: 50px;
+    box-sizing: border-box;
+    flex-shrink: 0;
+    height: @spacing-300;
+    line-height: 0;
+    margin-right: -8px;
+    padding: 0;
+    width: @spacing-300;
+}
+
+button.chip__button:focus-visible {
+    // we don't currently have an appropriate token for this
+    // all of the current tokens using the prop are for disabled states
+    background-color: var(--color-neutral-3);
+    outline-color: var(--color-state-focus-stroke);
+    outline-offset: -2px;
+    outline-style: solid;
+    outline-width: 2px;
+}
+
+button.chip__button:hover {
+    // we don't currently have an appropriate token for this
+    // all of the current tokens using the prop are for disabled states
+    background-color: var(--color-neutral-3);
+}
+
+button.chip__button:hover > svg {
+    fill: var(--color-foreground-on-primary);
+}
+
+[dir="rtl"] .chip__button {
```

Comment:
Should we make this new module our first use case for the new logical layout properties? 

https://github.com/eBay/skin/issues/1885

https://developer.mozilla.org/en-US/docs/Web/CSS/margin-block-start

Or wait and do it all in one go? I'm in favor of starting right here and using it as our first example, given that it's simple.

Reply:
I'm not familiar with `margin-block-start`, but browser compatibility on `dir` is pretty horrific. 
<img width="795" alt="image" src="https://github.com/eBay/skin/assets/1675667/4fb714ab-4fc9-4491-b241-0775c2498cdc">

Or would `margin-block-start` be a completely self-sustained solution not relying on things that have little to no support?

Reply:
Basically they are non-direction specific replacements for padding-left, padding-right, margin-left, margin-right, etc.

In a LTR language margin-block-start is equivalent to margin-left. In a RTL language it would be equivalent to margin-right. Therefore we no longer would need to explicitly override margin-left & right in an `[dir=rtl] { }` selector

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values